tools/replay/desktop_main.cpp

```diff
@@ -174,74 +176,85 @@ int main(int argc, const char** argv)
             if (dx_replay_options.enable_d3d12)
             {
                 application->InitializeDx12WsiContext();
-
-                dx12_replay_consumer.SetFatalErrorHandler(
-                    [](const char* message) { throw std::runtime_error(message); });
-                dx12_replay_consumer.SetFpsInfo(&fps_info);
-
-                // check for user option if first pass tracking is enabled
-                if (dx_replay_options.enable_d3d12_two_pass_replay)
+                if (gfxrecon::graphics::dx12::VerifyAgilitySDKRuntime() == false)
+                {
+                    GFXRECON_LOG_FATAL(
+                        "Did not find Agility SDK runtimes. Verify \\D3D12\\D3D12Core.dll exists in the same "
+                        "directory as gfxrecon-replay.exe.");
+                    return_code = -1;
```

Comment:
Having the Agility SDK runtimes is a hard requirement after we set 
`D3D12SDKVersion = 606` 
`D3D12SDKPath = u8".\\D3D12\\"`. 

Since we set that, I don't know that the app would have anywhere to find a runtime, so why continue? Also, the warning/error message would get lost in spam. Example, this fills the terminal window. So if we need to to continue we'd have to make sure subsequent errors don't drown out the warning/error.

`[gfxrecon] WARNING - Skipping memory fill for unrecognized mapped memory object (ID = 2496276791296)`